charts/cloudquery/templates/admin-deployment.yaml

```diff
@@ -33,12 +33,18 @@ spec:
           resources:
           {{- toYaml .Values.resources | nindent 12 }}
           volumeMounts:
+          {{- if .Values.volumeMounts }}
+          {{- toYaml .Values.volumeMounts | nindent 10 }}
```

Comment:
shouldn't it be 12 as in `resources` above?

Reply:
That doesn't work since there are already `volumeMounts` hardcoded. Doing it like `resources` results in.

```
volumeMounts:
null
- foo: bar
```

For this reason we need the check if supplied.